src/interfaces/address-interfaces.ts

```diff
@@ -1,23 +1,24 @@
-// Base address Interface to match the address.json schema and it's optional fields.
+/** Address interface to match the address.json schema plus optional fields. */
 export interface AddressIF {
-  actions?: string[];
-  addressCity: string;
-  addressCountry: string;
-  addressRegion: string;
-  deliveryInstructions?: string;
-  postalCode: string;
-  streetAddress: string;
-  streetAddressAdditional?: string;
+  actions?: string[]
+  addressCity: string
+  addressCountry: string
+  addressRegion: string
+  deliveryInstructions?: string
+  postalCode: string
+  streetAddress: string
+  streetAddressAdditional?: string
+  addressType?: 'mailing' | 'delivery'
```

Comment:
I added "addressType" so that I could maintain through the data and not have type errors.

Reply:
(ie, same as actions)

(Previously, sometimes addresses had actions or addressType and sometimes they didn't. Now they always do.)